test/unit/object/test_event.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+
+from __future__ import unicode_literals
+
+from mock import Mock
+import pytest
+
+from boxsdk.object.event import Event
+
+
+def test_init_event(mock_box_session):
+    event = Event(mock_box_session, "f82c3ba03e41f7e8a7608363cc6c0390183c3f83", {
+       "type": "event",
```

Comment:
formatting here... actually travis is probably complaining about this.
we usually use the format like:
event = Event(
  arg1,
  next_arg_on_new_line_if_we_are_using_multiline,
  {
    foo: bar,
    baz: wing,
    ding: {
      but_were_not_super_strict_I_didnt_put_the_above_curley_on_its_own_line: value1,
      but_we_are_picky_about_the_last_item_having_a_trailing_comma: value2,
    },
  },
)